closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -93,18 +91,11 @@ public final class RestrictedSecurity {
         supportedPlatformsOpenJCEPlus.put("Arch", List.of("amd64", "ppc64", "s390x"));
         supportedPlatformsOpenJCEPlus.put("OS", List.of("Linux", "AIX", "Windows"));
 
-        @SuppressWarnings("removal")
-        String[] props = AccessController.doPrivileged(
-                new PrivilegedAction<>() {
-                    @Override
-                    public String[] run() {
-                        return new String[] { System.getProperty("semeru.fips"),
+        String[] props = new String[] { System.getProperty("semeru.fips"),
                                 System.getProperty("semeru.customprofile"),
                                 System.getProperty("os.name"),
                                 System.getProperty("os.arch"),
                                 System.getProperty("semeru.fips.allowsetproperties") };
```

Comment:
Suggest we no longer want to use an array. Please use a separate local for each property as necessary.
```java
        String osName = System.getProperty("os.name");
        String osArch = System.getProperty("os.arch");
```
(replacing references to `props[2]` and `props[3]`). 
Lines 134-135 become
```java
        userEnabledFIPS = Boolean.getBoolean("semeru.fips");
        allowSetProperties = Boolean.getBoolean("semeru.fips.allowsetproperties");
```
Lines 145-150 become
```java
        selectedProfile = System.getProperty("semeru.customprofile");
        userSetProfile = selectedProfile != null;
```